indico/modules/bootstrap/controllers.py

```diff
@@ -44,14 +47,26 @@ class RHBootstrap(RH):
     def _process_GET(self):
         if User.query.filter_by(is_system=False).has_rows():
             return redirect(url_for_index())
+
+        try:
+            hostname = urlparse(request.url_root).hostname.strip('[]')
+            host_data = socket.getaddrinfo(hostname, 80, 0, 0, socket.IPPROTO_TCP)
+            for item in host_data:
```

Comment:
local_instance = any(ipaddress.ip_address(unicode(item[4][0])).is_private for item in host_data)